dts/bindings/pinctrl/raspberrypi,pico-pinctrl.yaml

```diff
@@ -121,3 +121,17 @@ child-binding:
         description: |
           The slew rate of a pin. 0 corresponds to slow, and 1 corresponds to fast.
           The default value is 0 (slow), as this is the power on reset value.
+      oe-override:
+        type: int
+        enum:
+          - 0
+          - 1
+          - 2
+          - 3
+        default: 0
+        description: |
+          Override output-enable for a pin.
+          0 - drive output enable from selected peripheral signal (default)
```

Comment:
justify default https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values